packages/liferay-js-insights/index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const globby = require('globby');
+const findUp = require('find-up');
+const fs = require('fs');
+const util = require('util');
+const argv = require('minimist')(process.argv.slice(2));
+const parser = require('@babel/parser');
+const path = require('path');
```

Comment:
Let me just sort these for you... I have a lint for that.

Reply:
😁